source/containers.tex

```diff
@@ -824,6 +903,23 @@
 \pnum
 The complexities of the expressions are sequence dependent.
 
+% Local command to index names as members of all containers.
+\renewcommand{\indexcont}[1]{%
+\indexlibrarymisc{#1}{sequence containers}%
+\indexlibrarymemberx{deque}{#1}%
+\indexlibrarymemberx{forward_list}{#1}%
+\indexlibrarymemberx{list}{#1}%
+\indexlibrarymemberx{vector}{#1}%
+\indexlibrarymemberx{map}{#1}%
+\indexlibrarymemberx{set}{#1}%
+\indexlibrarymemberx{multiset}{#1}%
+\indexlibrarymemberx{multimap}{#1}%
+\indexlibrarymemberx{unordered_map}{#1}%
+\indexlibrarymemberx{unordered_set}{#1}%
+\indexlibrarymemberx{unordered_multiset}{#1}%
+\indexlibrarymemberx{unordered_multimap}{#1}%
```

Comment:
Won't this index the sequence container members as members of these non-sequence containers?

Reply:
Yes. Fixed.